src/pages/VeSDL/LockedInfo.tsx

```diff
@@ -1,32 +1,68 @@
 import { Box, Paper, Typography } from "@mui/material"
-import React from "react"
-import { useTranslation } from "react-i18next"
+import React, { useEffect, useState } from "react"
+import {
+  useSdlContract,
+  useVotingEscrowContract,
+} from "../../hooks/useContract"
 
-type LockedInfoProps = {
-  sdlLocked?: string
-  totalveSdl?: string
-  avgLockTime?: string
-}
+import { BigNumber } from "@ethersproject/bignumber"
+import { ChainId } from "../../constants"
+import { formatBNToShortString } from "../../utils"
+import { useActiveWeb3React } from "../../hooks"
+import { useTranslation } from "react-i18next"
 
-export default function LockedInfo({
-  sdlLocked,
-  totalveSdl,
-  avgLockTime,
-}: LockedInfoProps): JSX.Element {
+// TODO add avgLockTime data
+export default function LockedInfo(): JSX.Element {
   const { t } = useTranslation()
+  const { account, chainId, library } = useActiveWeb3React()
+
+  const votingEscrowContract = useVotingEscrowContract()
+  const sdlContract = useSdlContract()
+  const [aggSDLInfo, setAggSDLInfo] = useState<{
```

Comment:
Does we need to have some agreement for camal case for `SDL` and `veSDL`.
For example, `aggSDLInfo` and `aggSdlInfo`.